packages/tpetra/core/test/Map/Map_Bug5378.cpp

```diff
@@ -145,10 +149,10 @@ TEUCHOS_UNIT_TEST( Map, Bug5378_BadGIDsNoLIDs )
   //
   // Lookup of any valid global ID should identify with proc 0
   //
-  RCP<const Tpetra::Map<int,long> > map = Tpetra::createContigMap<int,long>(10,10,comm);
-  Array<long> lookup_gids(  tuple<long>(1,10,5) );
-  Array<int> expected_ids(  tuple<int>( 0,-1,0) );
-  Array<int> nodeIDs( lookup_gids.size() );
+  RCP<const Tpetra::Map<LO,GO> > map = Tpetra::createContigMap<LO,GO>(10,10,comm);
+  Array<GO> lookup_gids(  tuple<GO>(1,10,5) );
+  Array<LO> expected_ids(  tuple<LO>( 0,-1,0) );
```

Comment:
Line 154: expected_ids should be int, not LO, because these are MPI process ranks.